### DISCUSSION TITLE
Ender 3 S1 with Pro Extruder

### DISCUSSION THREAD

**Discussion by MrGrendel27** [mriscoc/Ender3V2S1#123] [Q&A] [answered]
Hello, I have a stock Ender 3 S1 and I have just received the Sprite Pro Kit. Can this be a straight plug and play swap with extruders and print without any issues?  I'm hoping at some point to use some CF nylon so that's why I got this. What can/would need to be done to get the 300 C printing ability? I did install the pro firmware but was having problems with the bed leveling and getting the lines under the Z-offset on the display to turn blue meaning it was using the mesh. So that's why I'm back with the stock firmware for now.

Thanks for any help!

**Comment by mriscoc** [comment, marked as answer]
Check https://github.com/mriscoc/Special_Configurations/releases/T13

**Reply by MrGrendel27** [reply]
OK,
So I'm guessing I need the AM3950T13 config?? Two questions.
1. Will this work with the stock firmware? ( I do plan on giving the pro firmware a try again soon. I really did like it but had problems with bed level.)
2. How do I install this? Do I just put it on an SD card and turn the machine on and flash it like I did when I had the pro firmware?

Sorry I'm pretty new at this I've only been printing for about 2 months now and trying to learn as much as I can.

Thanks!!

**Reply by mriscoc** [reply]
No, the configuration files are for the most recent version of Marlin base code.

**Reply by MrGrendel27** [reply]
Ok.....now I'm lost. What are you suggesting I do to get to print to a higher temp ?

**Reply by mriscoc** [reply]
You can use the precompiled versions from the special configuration's repository https://github.com/mriscoc/Special_Configurations/releases

**Reply by t-g-b** [reply]
Hi  @MrGrendel27 ,
sorry for jumping in here and apologies for the beginner question. Did you have any luck getting the Pro Kit to work? Could you share what you changed? In the other issue it was mentioned that the Thermistor value has to be changed as well - correct?

EDIT: I found the configuration file. Any feedback if it worked or not would be helpful :-)

**Reply by MrGrendel27** [reply]
I had no idea of what to do with the configuration files. I reached out to a local filament company and made some connections and one of their tech guys rewrote the factory Creality firmware so I could print to 285C. So far its been working great!!

**Reply by t-g-b** [reply]
Thanks @MrGrendel27 .
In the meantime I tried the prebuilt firmware mentioned above (no need to change the configuration files). Many thanks @mriscoc for your efforts! It works well for me (Ender 3 S1 with Pro Extruder) so far.

**Reply by Skydsler** [reply]
hi @mriscoc ,

is it possible that you precompile a 300° pro extruder firmware for the F4 version S1?

**Reply by mriscoc** [reply]
> hi @mriscoc ,
> 
> is it possible that you precompile a 300° pro extruder firmware for the F4 version S1?

Check if that configuration was already sponsored here: https://github.com/mriscoc/Special_Configurations/releases

**Reply by Skydsler** [reply]
Hi @mriscoc . There are 2 problems... 
1st... In your main source is the fix for #125  not included. Can u provide the source with the fix?
Ive compiled it finaly and it will not save the settings after a power cycle.
2nd... While compiling with ur special config ( every ) i get an error in G28.cpp in line 130.
After deleting the constexpr for line 130 it will compile without a erreor..

**Reply by mriscoc** [reply]
The source will be available along with the release of the latest firmware version maybe this weekend.

**Reply by mm108** [reply]
> is it possible that you precompile a 300° pro extruder firmware for the F4 version S1?

hi @Skydsler, could you manage to get it to compile properly? If yes, is there a precompiled version that you could point me to? I am kinda new to tinkering with the firmware and so a little hesitant at the moment as I can't afford any long downtime for my S1 Pro should I brick it by any chance. My chip is a F4 version and currently I am running this non 300° C version. It works very well too but I need to print some PC filament which may require up to 290° C, Thanks a ton in advance. Even if you could point me to some detailed steps, that would be great too.

**Reply by mriscoc** [reply]
Go to assets at the end of https://github.com/mriscoc/Special_Configurations/releases/T13
